org.cockpit-project.certificates.metainfo.xml

```diff
@@ -2,15 +2,17 @@
 <component type="addon">
   <id>org.cockpit_project.certificates</id>
   <metadata_license>CC0-1.0</metadata_license>
-  <name>Starter Kit</name>
+  <name>Certificates</name>
   <summary>
-    Scaffolding for a cockpit module.
+    Manage your certificates.
   </summary>
   <description>
     <p>
-      Scaffolding for a cockpit module.
+      This tool manages certificates. With it, you can request, manage and delete certificates.
```

Comment:
This is rather redundant. Something like "You can request, view, refresh, and delete certmonger certificates" (not sure if "refresh" is true, but the UI did sound like that?

Reply:
One can configure autorenewal but I don't see how to request a 'refresh'.

Reply:
Then let's mention that instead (mine was just an example how to make this more concrete)